controllers/busRouteAssignController.js

```diff
@@ -0,0 +1,28 @@
+import { success, fail } from '../function/respond';
+import { buses as busModel, routes as routeModel } from '../models';
+
+
+export const Assign = async (req, res) => {
+	const { plate, code } = req.params;
+	const bus = await busModel.findOne({
+		where: {
+			platenumber: plate,
+		},
+	});
+	const route = await routeModel.findOne({
+		where: {
+			routecode: code,
+		},
+	});
+	if (!bus)
+		return fail(res,404,null,'busNotExist',req) 
+	if (!route)
+		return fail(res,404,null,'routeNotExist',req) 
+
+		
+	bus.routeId = route.id;
+	bus.routecode = route.code;
+	await bus.save();
+		return success(res,200,bus,'busAssignedSuccessfully',req) 
+
```

Comment:
- What happens when we assign a bus to another route when it's already assigned to a route? or
- How can I remove a bus from a specific route?

Reply:
on the first point, when you tempt to assign a bus to another route it reassigns meaning it updates the current data by removing current route and assign to new assigned route while on the second point I have added a new route to handle that.